src/redismodule.h

```diff
@@ -260,33 +277,86 @@ static const RedisModuleEvent
 #define REDISMODULE_SUBEVENT_PERSISTENCE_SYNC_RDB_START 2
 #define REDISMODULE_SUBEVENT_PERSISTENCE_ENDED 3
 #define REDISMODULE_SUBEVENT_PERSISTENCE_FAILED 4
+/* Next REDISMODULE_SUBEVENT_PERSISTENCE flag,
+ * this flag should no be used directly by the user.
+ * Instead, users should use RedisModule_IsSubEventSupported.
+ * When adding a new flag this value should be updated*/
```

Comment:
i would drop all these long comments. i think the `_` says enough, and if not, then a short comment on the same line of the define should be used..

IMHO that many comment lines limit the readability of the code, specially considering that it's repeated so many times.